chat-android/src/main/java/com/ably/chat/RoomOptions.kt

```diff
@@ -81,12 +81,18 @@ data class TypingOptions(
 /**
  * Represents the reactions options for a chat room.
  */
-object RoomReactionsOptions
+class RoomReactionsOptions {
```

Comment:
- Why don't we use empty `data class` instead?
- This will break roomOptions check -> CHA-RC1f1 
https://github.com/ably/ably-chat-kotlin/blob/6db96469b0108140c8892454dd10aa7e0b4bbc28/chat-android/src/main/java/com/ably/chat/Rooms.kt#L82
- Above simple check look for changed values instead of reference.

Reply:
Seems we can't create empty data class, but we can specify 
`
data class RoomReactionsOptions(val enabled : Boolean = true)
`
and
`
data class OccupancyOptions(val enabled : Boolean = true)
`
for now

Reply:
Tests for the same -> https://github.com/ably/ably-chat-kotlin/blob/3b98b7c300dd45b03503a6b2d1169f0648370a8d/chat-android/src/test/java/com/ably/chat/room/RoomGetTest.kt#L69
and 
https://github.com/ably/ably-chat-kotlin/blob/3b98b7c300dd45b03503a6b2d1169f0648370a8d/chat-android/src/test/java/com/ably/chat/room/RoomGetTest.kt#L47

Reply:
updated to 
```
data class RoomReactionsOptions(val logging : Boolean = true)
data class OccupancyOptions(val logging : Boolean = true)
```

Reply:
when we should refactor everything and use `enable` property instead of nulls. It's better to have single way to enable / disable options. 

 > This will break roomOptions check -> CHA-RC1f1

No it won't, I added `equals()` and `hashCode()` and I specifically added tests for this check

Reply:
It will break because `equals` checks for `reference` instead of values

Reply:
Okay, seems `equals` check for instance type so it doesn't care about reference

Reply:
So, in the future we can refactor this to data class and it should work without breaking public API.

Reply:
LGTM